person.rb

```diff
@@ -5,7 +5,7 @@ class Person < Nameable
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
-    super(name)
+    super
```

Comment:
- [ ] Kindly change this to `super()`. I could not run your app due to this. That should fix all your issues.

  Note that `super` and `super()` have very different applications. You can see [this StackOverflow comment](https://stackoverflow.com/a/31816281) for more description.